tests/UI/pages/BO/shopParameters/contact/add.js

```diff
@@ -43,8 +52,8 @@ class AddContact extends BOBasePage {
 
   /**
    * Fill form for add/edit contact
-   * @param page
-   * @param contactData
+   * @param page {Page} Browser tab
+   * @param contactData {object} Data to set on contact form
```

Comment:
```suggestion
   * @param contactData {ContactData} Data to set on contact form
```

Maybe we can link it after that if we add js-doc on faker data files

Reply:
Done tx :+1: